crates/nargo_cli/tests/execution_success/regression_mem_op_predicate/src/main.nr

```diff
@@ -0,0 +1,8 @@
+fn regression_mem_op_predicate(mut x: [u32; 5], idx: Field) {
```

Comment:
This needs to be called from a main func to work

Reply:
oops...!

may be it should fail if there is no main (nor contract or lib).